scripts/run-benchmarks.py

```diff
@@ -0,0 +1,185 @@
+import subprocess
+import json
+import os
+import re
+
+import click
+import cpuinfo
+
+BASE_REPO_URL = 'https://github.com/lumol-org/lumol.git'
+
+INDIVIDUAL_BENCH_TEMPLATE = """
+<details><summary>%s %s</summary>
+  <p>\n\n
+\n\n
+```bash\n
+%s\n\n
+```\n\n
+
+</p></details>
+"""
+
+COMPARISON_TEMPLATE = """
+<details><summary>%s %s</summary>
+  <p>\n\n
+\n\n
+```bash\n
+%s\n\n
+```\n\n
+
+</p></details>
+"""
+
+
+def request_api(endpoint, data=None):
+    url = 'https://api.github.com/repos/lumol-org/lumol' + endpoint
+    username = os.environ['LUMOL_GH_USERNAME']
+    token = os.environ['LUMOL_GH_TOKEN']
+
+    cmd = "curl -sS %s  -u '%s:%s'" % (url, username, token)
```

Comment:
Can we have a nice error message if curl is not found? 

Or if you have the motivation (I would understand if not ^^), could we use the `request` Python package here?